<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>291</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabBar {
	font: 700 9pt &quot;Segoe UI&quot;;
        alignment: center;
    }

QTabBar::tab {
        margin-left: auto;
        margin-right: auto;
        
	
	background-color: rgb(158, 105, 103);
        padding: 10px;
        border: 1px solid gray;
        margin-right: 2px;
		border-radius: 10px;
    }

   QTabBar::tab:selected {
        
	background-color: rgb(110, 70, 68);
        color: white;
        border-bottom: 2px solid blue;
    }

    QTabBar::tab:hover {
        background-color: lightblue;
    }</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_21">
     <attribute name="title">
      <string>Recherche</string>
     </attribute>
     <widget class="QTableView" name="tableView_7">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>301</width>
        <height>391</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
	background-color: rgb(110, 70, 68);
</string>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="rechercheB">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>330</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 700 9pt &quot;Segoe UI&quot;;
background-color: rgb(110, 70, 68);</string>
      </property>
      <property name="text">
       <string>Recherche</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rechidprod">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>9</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(158, 105, 103);
border-radius: 5px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_55">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>    Id Produit : </string>
      </property>
     </widget>
     <widget class="QTableView" name="affichage">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;background-color: rgb(158, 105, 103);&lt;/p&gt;&lt;p&gt;border-radius: 25px;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(158, 105, 103);
border-radius: 25px;
</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pdfB_R">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>330</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 700 9pt &quot;Segoe UI&quot;;
background-color: rgb(110, 70, 68);</string>
      </property>
      <property name="text">
       <string>PDF EXPORT</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_22">
     <attribute name="title">
      <string>Tri</string>
     </attribute>
     <widget class="QTableView" name="tableView_8">
      <property name="geometry">
       <rect>
        <x>-30</x>
        <y>0</y>
        <width>351</width>
        <height>391</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(110, 70, 68);
border-radius: 30px;</string>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="Line" name="line_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>301</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="trib">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>330</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 700 9pt &quot;Segoe UI&quot;;
background-color: rgb(110, 70, 68);</string>
      </property>
      <property name="text">
       <string>Tri</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxtri">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Date_ent</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Prix</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Qte</string>
       </property>
      </item>
     </widget>
     <widget class="QTableView" name="afftri">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;background-color: rgb(158, 105, 103);&lt;/p&gt;&lt;p&gt;border-radius: 25px;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(158, 105, 103);
border-radius: 25px;
</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>421</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabBar {
    font: 700 9pt &quot;Segoe UI&quot;;
    alignment: center;
}

QTabBar::tab {
    margin-left: auto;
    margin-right: auto;
    background-color: rgb(18, 54, 92); /* Default tab color */
    padding: 10px;
    border: 1px solid gray;
    margin-right: 2px;
    border-radius: 10px;
}

QTabBar::tab:selected {
    background-color: rgb(198, 1, 2);  /* Red background for selected tab */
    color: white;
    border: 2px solid darkred;         /* Darker border for emphasis */
    border-bottom: 3px solid blue;    /* Optional blue underline */
}

QTabBar::tab:hover {
    background-color: lightblue;       /* Highlight color when hovering over tab */
}
</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_25">
     <attribute name="title">
      <string>Ajout</string>
     </attribute>
     <widget class="QTableView" name="tableView_11">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>491</width>
        <height>381</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(187, 24, 28);
</string>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_52">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>0</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);

font: 700 15pt &quot;Segoe UI&quot;;
text-decoration: underline;</string>
      </property>
      <property name="text">
       <string>Ajouter stock </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_54">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>-ID PRODUIT </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_56">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>-DESIGNATION</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_57">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>-PRIX</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_58">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>-ETAT</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_59">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>-QUANTITE</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_60">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>-DATE ENTREE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_15">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 700 9pt &quot;Segoe UI&quot;;
background-color: rgb(79, 10, 13);</string>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_16">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>330</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 700 9pt &quot;Segoe UI&quot;;
background-color: rgb(79, 10, 13);</string>
      </property>
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Idprod">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>231</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="des">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>231</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="prix">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>120</y>
        <width>231</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="qte">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>160</y>
        <width>231</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="etat">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>200</y>
        <width>231</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_61">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>-DATE EXP</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouterB">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>260</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 700 9pt &quot;Segoe UI&quot;;
background-color: rgb(110, 70, 68);</string>
      </property>
      <property name="text">
       <string>ajouter</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_ent">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>240</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_exp">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>270</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_28">
     <attribute name="title">
      <string>modification</string>
     </attribute>
     <widget class="QTableView" name="tableView_12">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>-10</y>
        <width>441</width>
        <height>381</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(187, 24, 28);
</string>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_53">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>0</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);

font: 700 15pt &quot;Segoe UI&quot;;
text-decoration: underline;</string>
      </property>
      <property name="text">
       <string>Modifier stock</string>
      </property>
     </widget>
     <widget class="QComboBox" name="boxmodif">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>120</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Designation</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Prix</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>qte</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Etat</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Date_ent</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Date_exp</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_65">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>-ID PRODUIT </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="modidprod">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>60</y>
        <width>231</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="mod">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>120</y>
        <width>231</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ModifierB">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>210</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 700 9pt &quot;Segoe UI&quot;;
background-color: rgb(110, 70, 68);</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_26">
     <attribute name="title">
      <string>Suppression</string>
     </attribute>
     <widget class="QTableView" name="tableView_13">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>-10</y>
        <width>441</width>
        <height>381</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(187, 24, 28);
</string>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_62">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>0</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);

font: 700 15pt &quot;Segoe UI&quot;;
text-decoration: underline;</string>
      </property>
      <property name="text">
       <string>Supprimer stock</string>
      </property>
     </widget>
     <widget class="QPushButton" name="supprimerB">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>110</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 700 9pt &quot;Segoe UI&quot;;
background-color: rgb(110, 70, 68);</string>
      </property>
      <property name="text">
       <string>supp</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="suppidstock">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>231</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_64">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 10pt &quot;Segoe UI&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>    Id Produit : </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_27">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>327</height>
      </size>
     </property>
     <attribute name="title">
      <string>statistiques</string>
     </attribute>
     <widget class="QTableView" name="tableView_14">
      <property name="geometry">
       <rect>
        <x>-20</x>
        <y>-10</y>
        <width>441</width>
        <height>381</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(187, 24, 28);
</string>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_63">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);

font: 700 15pt &quot;Segoe UI&quot;;
text-decoration: underline;</string>
      </property>
      <property name="text">
       <string>STAT</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="grafetat">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>421</width>
        <height>321</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(187, 24, 28);
</string>
      </property>
     </widget>
     <widget class="QPushButton" name="afffstat_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>250</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 700 9pt &quot;Segoe UI&quot;;
background-color: rgb(110, 70, 68);</string>
      </property>
      <property name="text">
       <string>stat</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pdfB">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>250</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 700 9pt &quot;Segoe UI&quot;;
background-color: rgb(110, 70, 68);</string>
      </property>
      <property name="text">
       <string>PDF EXPORT</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="sendServoCommand">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 700 9pt &quot;Segoe UI&quot;;
background-color: rgb(110, 70, 68);</string>
    </property>
    <property name="text">
     <string>PDF EXPORT</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>959</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
